test/mock-child_process.ts

```diff
@@ -10,30 +10,24 @@ export interface Invocation {
   cwd?: string;
   exitCode?: number;
   stdout?: string;
-
-  /**
-   * Only match a prefix of the command (don't care about the details of the arguments)
-   */
```

Comment:
Why are you removing comments?

Reply:
I thought it was a bit silly but I'll add it back